akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -94,6 +94,8 @@ The source code of this sample can be found in the
   [Akka Cluster Bootstrap](https://developer.lightbend.com/docs/akka-management/current/bootstrap.html) module.
 @@@
 
+### Manually joining to seed nodes
```

Comment:
Maybe call this Joining configured seed nodes; manually can mean many things

Reply:
Sure, updated.